spec/factories/transactions.rb

```diff
@@ -2,9 +2,8 @@
 
 FactoryBot.define do
   factory :transaction do
+    transactinable { |transaction| transaction.association(:balance_transaction) }
     amount { Random.rand(100) }
```

Comment:
@sinpai can't `Faker` be used for a random number? We use `Faker` everywhere, will be cool to be consistent

Reply:
```ruby
Faker::Number.digit.to_i
```
?